### DISCUSSION TITLE
Plan to incorporate landuse change emission

### DISCUSSION THREAD

**Discussion by arby1005** [USEPA/supply-chain-factors#35] [Q&A]
Hi,
SBTi has published the new FLAG guidance, which means the land use change emission should be calculated separately. Besides, the FLAG target incorporates also land management emissions, for example, the enteric emission from cattle grazing. 

Currently, the EEIO does not cover the land use change emission, and the land management emissions are integrated with the other upstream process. 

In order to set a FLAG and non-FLAG target, is there any plan in the future to publish a separate set of EEIO emission factors for land use change and land management emissions?

Thank you!

**Comment by WesIngwersen** [comment]
@arby1005 Thanks for raising this. Can you please add links to the institutions and guidance docs for my and others benefit?

**Reply by arby1005** [reply]
Hi @WesIngwersen, Thank you for the response. 
This is the link to the Science-based target website: https://sciencebasedtargets.org/
It is an initiative that drives ambitious climate action in the private sector by enabling organizations to set science-based emissions reduction targets.
And this is the link to the FLAG (Forest, Land and Agriculture) guidance mentioned: https://sciencebasedtargets.org/resources/files/SBTiFLAGGuidance.pdf
It is a guidance to set science-based targets that apply to a company’s GHG emissions from AFOLU, including GHG emissions associated with land use change (LUC); emissions from land management; and biogenic removals.

**Reply by arby1005** [reply]
Hi all,
Just wondering if anyone has some thoughts on this. It has always been a struggle to calculate the land use change emission when data granularity is not good enough.

**Reply by arby1005** [reply]
Hi all,
Just wondering if anyone has some thoughts on this.?
